resolwe_bio/processes/alignment/star.yml

```diff
@@ -144,6 +150,58 @@
           required: false
           description: >
             Maximum gap between two mates, if 0, max intron gap will be determined by (2pow(winBinNbits)*winAnchorDistNbins) (default: 0).
+        - name: alignEndsType
+          label: --alignEndsType
+          type: basic:string
+          required: false
+          description: >
+            Type of read ends alignment (default: Local).
+          default: Local
+          choices:
+            - label: Local
+              value: Local
+            - label: EndToEnd
+              value: EndToEnd
+            - label: Extend5pOfRead1
+              value: Extend5pOfRead1
+            - label: Extend5pOfReads12
+              value: Extend5pOfReads12
+    - name: output_sam_bam
```

Comment:
This `.sam` output is not saved with `re-save-file`. When saving, first check if the file was crated. Write a test.

Reply:
Of course it is. By STAR's defaults the core resulting file from alignment is `Aligned.out.sam`. In our process we set `--outSAMtype BAM SortedByCoordinate`, which causes the same file to be named `Aligned.sortedByCoord.out.bam`.